src/command_modules/azure-cli-interactive/azure/cli/command_modules/interactive/azclishell/util.py

```diff
@@ -55,17 +55,12 @@ def parse_quotes(cmd, quotes=True, string=True):
     """ parses quotes """
     import shlex
 
-    if quotes:
-        args = shlex.split(cmd)
-    else:
+    try:
+        args = shlex.split(cmd) if quotes else cmd.split()
+    except ValueError:
         args = cmd.split()
```

Comment:
probably better to log the error

Reply:
The error looks ok, however, this interprets a single '\' as equivalent to '\\'.
This may be a problem if say, someone ends their input with '\' and it gets interpreted as part of an argument value.
The real problem is that the shell interprets the single '\' as an escape character and doesn't find something to escape.

This will also result in a different split than if using `shlex`, which may (or may not) have untended consequences